kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/ListerWatcher.java

```diff
@@ -29,7 +29,7 @@
 public interface ListerWatcher<T, L> {
   Watch watch(ListOptions params, Watcher<T> watcher);
 
-  L list(ListOptions params);
+  L list();
```

Comment:
Isn't this a breaking change?

Reply:
No this should not be considered a public facing interface.